beam-migrate/Database/Beam/Migrate/Backend.hs

```diff
@@ -95,8 +97,7 @@ data BeamMigrationBackend be m where
 -- | Monomorphic wrapper for use with plugin loaders that cannot handle
 -- polymorphism
 data SomeBeamMigrationBackend where
-  SomeBeamMigrationBackend :: ( BeamMigrateSqlBackend be
-                              , Typeable be )
+  SomeBeamMigrationBackend :: Typeable be
```

Comment:
Can you put the `BeamMigrateSqlBackend` constraint back?

Reply:
It's not needed because you get it when you unpack the thing inside.